mixer/adapter/prometheus/config/config.proto

```diff
@@ -159,4 +160,32 @@ message Params {
     // The set of metrics to represent in Prometheus. If a metric is defined in Istio but doesn't have a corresponding
     // shape here, it will not be populated at runtime.
     repeated MetricInfo metrics = 1;
+
+    // Describes the expiration policy for metrics generated by a prometheus handler.
+    //
+    // Example: A Metrics Expiration Policy of `{ metrics_expiry_duration: "10m", expiry_check_interval_duration: "1m" }`
+    // would configure the handler to delete all metrics that have received no updtaes for 10 minutes. Metrics would be checked
+    // every minute to determine whether or not they should be expired.
+    message MetricsExpirationPolicy {
+        // Required. Describes the desired lifetime of a metric. If the metric is not updated at any point during this duration, it
+        // will be removed from the set of metrics exported by the handler.
+        google.protobuf.Duration metrics_expiry_duration = 1 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true] ;
+
+        // Required. Describes the interval in which metrics will be checked to see if they have been stale for longer that the configured
+        // `metrics_expiry_duration`. This should be generally be set to a duration much shorter than the configured `metrics_expiry_duration`.
+        google.protobuf.Duration expiry_check_interval_duration  = 2 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true];
```

Comment:
Does this need to be configurable ?

Reply:
i think so. I'll add a default value (so it isn't required), but this limits the overall +/- on expiry time observed -- and that seems worthy of allowing configuration.